CMake/android_config.cmake

```diff
@@ -7,23 +7,21 @@ macro(os_set_flags)
     unset(APPLE)
     set(BUILD_WITH_TM2 OFF)
     set(BUILD_WITH_OPENMP OFF)
-    set(FORCE_LIBUVC ON)
+
     set(BUILD_GRAPHICAL_EXAMPLES OFF)
     set(ANDROID_STL "c++_static")
     set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -fPIC -pedantic -g -D_BSD_SOURCE")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -pedantic -g -Wno-missing-field-initializers")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-switch -Wno-multichar")
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fPIE -pie")
-    set(BACKEND RS2_USE_V4L2_BACKEND)
-    include(CMake/external_libusb.cmake)
-    set(LIBUSB1_LIBRARIES ${LIBUSB1_LIBRARIES} log)
+	set(FORCE_LIBUVC OFF)
+	set(BACKEND RS2_USE_ANDROID_BACKEND)
+	set(HWM_OVER_XU ON)
 endmacro()
 
 macro(os_target_config)
     if(BUILD_SHARED_LIBS)
         find_library(log-lib log)
         target_link_libraries(${LRS_TARGET} PRIVATE log)
```

Comment:
I know you didn't add this line in but I get a bunch of warnings saying `clang++: warning: -llog: 'linker' input unused [-Wunused-command-line-argument]` when I build. Judging from [the android docs](https://developer.android.com/studio/projects/configure-cmake#add-ndk-api) I think this should be
`target_link_libraries(${LRS_TARGET} PRIVATE ${log-lib})`

Reply:
I've fixed this line in 94423935503b2971f97f6d38c8c6fb856f485b29 as you're right about the docs.

But the warning you have is down to a "misconfiguration" in the toolchain file Conan is using (check `/opt/conan/android.toolchain.cmake` in your Android container). `llog` is added as a compiler rather than a linker flag. I've added fixing that as a todo for down the line :)